circle.yml

```diff
@@ -13,7 +13,10 @@ dependencies:
     - mix compile
 test:
   override:
-    - mix test --include requires_env --cover
+    - |
+      if [ ${CIRCLE_PR_USERNAME} ]; then MIX_ENV=test mix test --cover; else MIX_ENV=test mix test --include requires_env --cover; fi
```

Comment:
@SViccari the test suite can't run the same way for everyone. See https://circleci.com/docs/fork-pr-builds/#security-implications-of-running-builds-for-pull-requests-from-forks for discussion of security implications of leaking sensitive environment variables.